src/20250421_Multi_GHOGnosisLaunch/utils/GHOLaunchConstants.sol

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/**
+ * @title GHOLaunchConstants
+ * @notice Library containing all constants used across the GHO Gnosis Launch proposal
+ */
+library GHOLaunchConstants {
+  // Common Addresses
+  address internal constant RISK_COUNCIL = 0x8513e6F37dBc52De87b166980Fa3F50639694B60;
+
+  // CCIP Rate Limits
+  uint128 internal constant CCIP_RATE_LIMIT_CAPACITY = 1_000_000e18;
+  uint128 internal constant CCIP_RATE_LIMIT_REFILL_RATE = 200e18;
+  uint128 internal constant CCIP_BUCKET_CAPACITY = 15_000_000e18;
```

Comment:
Please use new CCIP rate recommendation from ChaosLabs: https://governance.aave.com/t/arfc-gho-cross-chain-parameter-adjustments/21950

Reply:
Also, capacity cannot be 15M because bridgeLimit on Ethereum is 20M. Bridge transactions will fail

Reply:
I do not understand why it can't be 15M. And in that case it must be 20M?